src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertManufacturer = "INSERT INTO manufacturer (name,country) values(?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement
+                        = connection.prepareStatement(insertManufacturer,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1,manufacturer.getName());
+            createManufacturerStatement.setString(2,manufacturer.getCountry());
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1,Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException("Can't create new manufacturer",e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't create new manufacturer " + manufacturer,e);
```

let's make exception messages more informative